### DISCUSSION TITLE
GitLab Premium with Four Keys

### DISCUSSION THREAD

**Discussion by beaujolaisruben** [GoogleCloudPlatform/fourkeys#424] [Q&A]
Hello,
My company is using GitLab PREMIUM. For example GitLab DORA API is only available with GitLab ULTIMATE.
Hence, could you tell me if Four Keys project can work with projects hosted within GitLab PREMIUM.
Thank you very much.

**Comment by colinfrei** [comment]
We're using Gitlab Enterprise, which afaict is pretty comparable, and it seems to work.
The Webhooks, which are the main feature, are available for all tiers: https://docs.gitlab.com/ee/user/project/integrations/webhooks.html

**Comment by sebrochette** [comment]
@beaujolaisruben: did you go forward with the Four Keys project implementation and were you successful with Gitlab?
